app/graphql/types/locale_enum.rb

```diff
@@ -1,7 +1,7 @@
 module Types
   class LocaleEnum < Types::BaseEnum
-    FastGettext.human_available_locales.each do |description, locale|
-      value locale, description: description
+    FastGettext.default_available_locales.each do |locale|
+      value locale
```

Comment:
Honestly, I am not sure if dropping description is okay here, @timogoebel. But language name is not available at this point, I can put locale name into description if needed (e.g. "en").

Reply:
maybe @xprazak2 could be of help here?

Reply:
Sorry for late reply, adding name into description might be safer if someone relies on description being present.

Reply:
Cool cool, so the reply is? :-)

Reply:
I believe the reply is:
```suggestion
      value locale, description: locale
```

Reply:
Amended this change.